app/views/comments/_comment.html.erb

```diff
@@ -11,14 +11,29 @@
     </div>
     <div>
       <% if current_user == comment.author %>
-        <%= link_to edit_comment_path(comment), class: "btn btn-link btn-sm text-muted" do %>
+        <%= link_to edit_comment_path(comment), remote: true, class: "btn btn-link btn-sm text-muted" do %>
           <i class="fas fa-edit fa-fw"></i>
         <% end %>
 
-        <%= link_to comment, data: { turbo_method: :delete }, class: "btn btn-link btn-sm text-muted" do %>
+        <%= link_to comment, method: :delete, class: "btn btn-link btn-sm text-muted", remote: true do %>
           <i class="fas fa-trash fa-fw"></i>
         <% end %>
       <% end %>
     </div>
   </div>
 </li>
+
```

Comment:
This script is not necessary because the link_to helper method takes care of breaking the link!